packages/sdk/src/utils/contracts.ts

```diff
@@ -165,19 +165,19 @@ export const getBridgeAdapters = (
   }
 ): BridgeAdapters => {
   const adapterData: BridgeAdapterData = {
-    ...(CONTRACT_ADDRESSES[l2ChainId]
+    ...(CONTRACT_ADDRESSES[l2ChainId] || opts?.contracts?.l1?.L1StandardBridge
       ? {
           Standard: {
             Adapter: StandardBridgeAdapter,
             l1Bridge:
-              opts.contracts?.l1?.L1StandardBridge ||
+              opts?.contracts?.l1?.L1StandardBridge ||
```

Comment:
```suggestion
              opts?.contracts?.l1?.L1StandardBridge ??
```
nit.  not at all important in this case just one of those random things I can't help but nit about